fundamentals/installation.md

```diff
@@ -15,6 +15,20 @@ This tutorial will teach you how to do a clean install of **napari**. It is aime
 $ pip install napari
 ```
 
+### choosing differnt backends
+The default napari backend is [PySide2](https://wiki.qt.io/Qt_for_Python), which is fine for most users. A small percentage of users may wish to use a [PyQt](https://www.riverbankcomputing.com/software/pyqt/intro) backend instead. 
+If you want to install napari with a [PyQt](https://www.riverbankcomputing.com/software/pyqt/intro) backend with a single line command, you can use:
+```
+pip install pyqt5 napari
+```
+
+Note: the order of installation is important, [pyqt5](https://pypi.org/project/PyQt5/) must be installed *before* napari. During installation napari checks whether [pyqt5](https://pypi.org/project/PyQt5/) is already available. 
+If  so, napari will use a [PyQt](https://www.riverbankcomputing.com/software/pyqt/intro) backend. 
+If there is no pre-existing installation of [pyqt5](https://pypi.org/project/PyQt5/), then napari will install and use 
+a [PySide2](https://wiki.qt.io/Qt_for_Python) backend. 
+This is the default, and fine for most users. 
```

Comment:
```suggestion
**Note**: the order of the packages in the above line is important, because [pyqt5](https://pypi.org/project/PyQt5/) must be installed *before* napari.
```